manpages.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+var manpagesMatcher = regexp.MustCompile(`\b(\w+)\((\d[\da-z_-]*)\)(\W|$)`)
```

Comment:
We should add a comment what this shall match, just that later readers of the code don't have to compile the Regex in their head to understand its purpose.